### PR TITLE
Add /biome/verify route

### PR DESCRIPTION
**Pull request by shannynalayna**
Adds a /biome/verify REST endpoint to verify a user's password.
Also documents this endpoint in the openapi spec.

Signed-off-by: Shannyn Telander <telander@bitwise.io>

### CODE REVIEW

**Comment by shannynalayna** [comment]
@RyanLassigBanks My understanding is that it's necessary in the case of having some client resource that is password protected. This verify route is specific to just verifying the password, as the login route deals out a token for that user.

File: libsplinter/src/biome/rest_api/actix/verify.rs [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                    AuthorizationResult::Failed => {
+                        error!("Failed to authorize user");
+                            HttpResponse::InternalServerError()
+                                .json(ErrorResponse::internal_error())
+                                .into_future()
+                    }
+                }
+            }))
+        })
+}
```
Comment (ltseeley) [inline libsplinter/src/biome/rest_api/actix/verify.rs]: Please add some tests if possible; see the the circuit/proposal read tests for examples of how to test endpoints: https://github.com/Cargill/splinter/blob/master/libsplinter/src/admin/rest_api/actix/circuits.rs

**Comment by shannynalayna** [comment]
Closing this story based on the discussion that this functionality is redundant and not necessary for what is currently implemented.

File: libsplinter/src/biome/rest_api/actix/verify.rs (hunk 1 above)
Comment (shannynalayna) [reply libsplinter/src/biome/rest_api/actix/verify.rs]: As it stands, there is a lot of background work that needs to go into setting up the unit tests. It involved editing the existing *-stores and how they're passed around to make the biome resources. There is a story to add in these tests: https://zenhub.cglcloud.com/app/workspaces/product-core-5d37298b0add7306b30efde2/issues/digitalsupplychain/product-core/2019

**Pull request merged**